### PR TITLE
Include 'triggering_switch' in combo switch events

### PR DESCRIPTION
**Pull request by avanwinkle**
This PR adds a new kwarg, `triggering_switch`, to events posted by a ComboSwitch device.

When the player enters a combination switch, e.g. hold one flipper and press the other, it can be useful to know _which_ switch was hit when. Currently this requires quite a bit of logic to implement.

With this PR, the combo events include a kwarg `triggering_switch` that declares the `tag:` group number of the switch that caused the event.

For example:
```
combo_switches:
  both_flippers:
    tag_1: left_flipper
    tag_2: right_flipper
    events_when_both: flipper_cancel
```

Player holds left flipper, presses right flipper:
```
Event: ======'both_flippers'====== Args={'triggering_switch': 2}
```
Player releases left flipper, still holding right:
```
Event: ======'both_flippers_one'====== Args={'triggering_switch': 1}
```

This change makes it easy for game logic to perform actions based on combo switches, for example rotating selections left or right by holding one flipper and pressing the other, or triggering event based on which direction an orbit shot was completed in.

### CODE REVIEW

**Comment by jabdoa2** [comment]
Good one!

File: mpf/devices/combo_switch.py [hunk 1]
```diff
@@ -247,7 +247,7 @@ def _switch_state(self, state):
         self.debug_log("New State: %s", state)
 
         for event in self.config['events_when_{}'.format(state)]:
-            self.machine.events.post(event)
+            self.machine.events.post(event, triggering_switch=triggering_switch)
```
Comment (jabdoa2) [inline mpf/devices/combo_switch.py]: We could also include the switch name here.

**Review by jabdoa2 — APPROVED** [review]

File: mpf/devices/combo_switch.py (hunk 1 above)
Comment (avanwinkle) [reply mpf/devices/combo_switch.py]: Updated! Now includes `triggering_group` with the group number and `triggering_switch` with the switch name.

**Comment by avanwinkle** [comment]
Thanks for the improvements!

**Pull request merged**